graphrag_sdk/kg.py

```
@@ -57,6 +66,44 @@ def __init__(
         self._model_config = model_config
         self.sources = set([])
 
+        if cypher_system_instruction is None:
+            cypher_system_instruction = CYPHER_GEN_SYSTEM
+        else:
+            if "{ontology}" not in cypher_system_instruction:
+                warnings.warn("Cypher system instruction should contain {ontology}", category=UserWarning)
```

Ruff (0.7.0)

No explicit `stacklevel` keyword argument found

(B028)